capstone-ui/src/Components/ResultsBox/ResultsBox.jsx

```diff
@@ -47,9 +47,20 @@ export default function ResultsBox( { locationLevel, countyFIPS } ) {
         }
         else if (locationLevel === 3) {
             data = electionResults.data.races.find( (race) => race.state_slug === stateName ).counties.find((county) => (county.fips%1000) === (countyFIPS%1000))
+
+            for (let candidate in data.results) {
```

Comment:
Do you have a limit to how much candidate results you would want to render in dialog? I expect not too many candidates per county, but to future proof a really long hover dialog with many candidates, you probably can add a max cap to how many candidates you find or use to build results to display.

Reply:
Not currently, but I do want to limit it and might take a look at it when I'm working on generating the candidate views.